nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/data/explorer/IngestionDataFormat.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Since java kusto sdk IngestionProperties didnt have description required for the NIFI processor attributes, hence I added it. But I agree it will be difficult to maintain in two places, so adding the description inline to the Nifi Processor allowable values attributes definition.